### PR TITLE
powm1

### PR DESCRIPTION
**Pull request by jtlap**
just an ulp change

### CODE REVIEW

File: test/unit/module/real/math/powm1.cpp [hunk 1]
```diff
@@ -30,14 +31,13 @@ EVE_TEST_TYPES( "Check return types of powm1"
...
   TTS_EXPR_IS( eve::powm1(T(), v_t()), T);
-//  TTS_EXPR_IS( eve::powm1(v_t(), T()), T);
 };
 
 //==================================================================================================
 // powm1  tests
 //==================================================================================================
 EVE_TEST( "Check behavior of powm1 on wide"
-        , eve::test::simd::restricted::ieee_reals
+        , eve::test::simd::ieee_doubles
```
Comment (jfalcou) [inline test/unit/module/real/math/powm1.cpp:40]: WHy the change ?

File: test/unit/module/real/math/powm1.cpp [hunk 2]
```diff
@@ -48,9 +48,9 @@ EVE_TEST( "Check behavior of powm1 on wide"
   using eve::detail::map;
   using v_t = eve::element_type_t<T>;
 
-  TTS_ULP_EQUAL(eve::powm1(a0, a1)      , map([](auto e, auto f) -> v_t { return std::pow(double(e), double(f))-1; }, a0, a1), 2);
-  TTS_ULP_EQUAL(eve::powm1(a2, a3)      , map([](auto e, auto f) -> v_t { return std::pow(double(e), double(f))-1; }, a2, a3), 5);
-  TTS_ULP_EQUAL(eve::diff_1st(eve::powm1)(a0, a1), eve::pow(a0, eve::dec(a1))*a1, 2);
+  TTS_RELATIVE_EQUAL(eve::powm1(a0, a1)      , map([](auto e, auto f) -> v_t { return eve::pow(double(e), double(f))-1; }, a0, a1), 100*eve::eps(eve::as<v_t>()));
+  TTS_RELATIVE_EQUAL(eve::powm1(a2, a3)      , map([](auto e, auto f) -> v_t { return eve::pow(double(e), double(f))-1; }, a2, a3), 100*eve::eps(eve::as<v_t>()));
+//  TTS_ULP_EQUAL(eve::diff_1st(eve::powm1)(a0, a1), eve::pow(a0, eve::dec(a1))*a1, 2);
```
Comment (jfalcou) [inline test/unit/module/real/math/powm1.cpp:53]: Delete don't comment

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: test/unit/module/real/math/powm1.cpp (hunk 2 above)
Comment (jtlap) [reply test/unit/module/real/math/powm1.cpp]: removed

File: test/unit/module/real/math/powm1.cpp (hunk 1 above)
Comment (jtlap) [reply test/unit/module/real/math/powm1.cpp]: removed

**Review by jfalcou — APPROVED** [review]

**Pull request merged**